### PR TITLE
Redshift patch

### PR DESCRIPTION
**Pull request by thoffmann-sf**
Added all RedShift datatypes to the Ghost_Records_per_Datatype macro and fixed the other macros that were broken for RedShift and Postgres

### CODE REVIEW

**Comment by tkiehn** [comment]
Hi Tim,

I think your branch contains the log file that got deleted with #169.

Best Regards,
Theo

File: macros/supporting/ghost_record_per_datatype.sql [hunk 1]
```diff
@@ -347,43 +347,50 @@
...
-        {%- elif datatype == 'CHARACTER' %} '{{unknown_value__STRING}}' as {{ alias }}
-        {%- elif datatype == 'INT' %} CAST('0' as INT) as {{ alias }}
-        {%- elif datatype == 'INT2' %} CAST('0' as INT2) as {{ alias }}
-        {%- elif datatype == 'INT8' %} CAST('0' as INT8) as {{ alias }}
-        {%- elif datatype == 'NUMERIC' %} CAST('0' as NUMERIC) as {{ alias }}
-        {%- elif datatype == 'FLOAT4' %} CAST('0' as FLOAT4) as {{ alias }}
-        {%- elif datatype == 'FLOAT' %} CAST('0' as FLOAT) as {{ alias }}
-        {%- elif datatype == 'BOOLEAN' %} CAST('FALSE' as BOOLEAN) as {{ alias }}
-        {%- elif datatype == 'VARBINARY' %} 'NULL'::varbyte as {{ alias }}
+        {%- elif datatype == 'VARCHAR' or datatype == 'CHARACTER VARYING' or datatype == 'NVARCHAR' or datatype == 'TEXT' %} '{{unknown_value__STRING}}' as {{ alias }}
```
Comment (tta-scalefree) [inline macros/supporting/ghost_record_per_datatype.sql:367]: please group the datatypes into array for ease of extending this code in the future, see line 363. 
Same for the following code lines.

**Review by tta-scalefree — COMMENTED** [review]
please review macro for ghost_record_per_datatype and align code with other adapters.

**Comment by tkiehn** [comment]
Hi,

the change from #172 did change the default global variable, which (if copied from the package) is used by all adapters.
I have opened PR #177 that changes the hardcoded default values directly in the macros, which are used if no global variable is specified.

Best Regards,
Theo

**Comment by tkirschke** [comment]
Hi @thoffmann-sf , 
Can you please add all related issues as related items to this PR?

Best regards
Tim

**Pull request merged**